launch/main_launch.py

```diff
@@ -0,0 +1,145 @@
+"""Launch file that runs all nodes for the boat simulator ROS package."""
+
+import importlib
+import os
+from typing import List
+
+from launch_ros.actions import Node
+
+import boat_simulator.common.constants as Constants
+from launch import LaunchDescription, LaunchDescriptionEntity
+from launch.actions import DeclareLaunchArgument, OpaqueFunction
+from launch.launch_context import LaunchContext
+from launch.substitutions import LaunchConfiguration
+
+# Local launch arguments and constants
+PACKAGE_NAME = "boat_simulator"
+
+# Add args with DeclareLaunchArguments object(s) and utilize in setup_launch()
+LOCAL_LAUNCH_ARGUMENTS = [
+    DeclareLaunchArgument(
+        name="enable_sim_multithreading",
+        default_value="false",
+        choices=["true", "false"],
+        description="Enable multithreaded execution of callbacks in the boat simulator",
+    )
+]
+
+# Add vars with SetEnvironmentVariable object(s)
+LOCAL_ENVIRONMENT_VARS = []
```

Comment:
We should probably remove this in all the package launch files, there is no such thing as a local environment variable. This could create undefined behavior if different packages define different values for the same environment variable

Reply:
Resolved in following commit